src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -118,6 +108,10 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
 
   toPreviousPage() {
     let caseEventData: CaseEventData = this.buildCaseEventData();
+    // caseEventData.event_data contains all the values from the previous pages so we set caseEventData.data = caseEventData.event_data
+    // This builds the form with data from the previous pages
+    // EUI-3732 - Breathing space data not persisted on Previous button click with ExpUI Demo
+    caseEventData.data = caseEventData.event_data;
```

Comment:
I wonder whether this comment could be put on top of the method in a proper JSdoc comment

Reply:
Good call. :)